drivers/gyro/adxrs290/adxrs290.c

```diff
@@ -292,6 +292,13 @@ int32_t adxrs290_init(struct adxrs290_dev **device,
 	if (IS_ERR_VALUE(ret))
 		goto error_spi;
 
+	// Set GPIO sync pin. Set adxrs290 to output on sync pin.
+	gpio_get(&dev->gpio_sync, &init_param->gpio_sync);
+	gpio_direction_input(dev->gpio_sync);
+	ret |= adxrs290_reg_write(dev, ADXRS290_REG_DATA_READY, ADXRS290_DATA_RDY_OUT);
```

Comment:
is this related to this commit?

Reply:
Yes. The GPIO needs to be initialized as input and the device should be configured to output the DATA_READY on the sync pin. This DATA ready is needed so that it can be used in the filling of buffers in the iio_demo in the next commits.